content/cloud/linking-applications-to-resources-using-labels.md

```diff
@@ -0,0 +1,152 @@
+
+```toml
+// --snip--
+name = "todo-app-v2"
+[component]
+id = "todo"
+sqlite_databases = ["default"]
+// --snip--
+```
+
+When deploying the second Spin application (`todo-app-v2`) we are again asked if we would like to use an existing database and link our application (in this case `todo-app-v2`) to it:
+
+```bash
+spin deploy       
+Uploading todo-app-v2 version 0.1.0-r6e577b18 to Fermyon Cloud...
+Deploying...
+App "todo-app-v2" accesses a database labeled "default"
+Would you like to link an existing database or create a new database?:
+> Use an existing database and link app to it
+  Create a new database and link the app to it
+```
+
+We again use the up/down arrow keys to make our selection and hit `return` to continue:
+
+```bash
+Which database would you like to link to todo-app-v2 using the label "default":
+> inspiring-otter
+```
+
+In the example here we have chosen to link `todo-app-v2` to our pre-existing `inspiring-otter` Cloud database using via the "default" label, that we added to our `todo-app-v2` application manifest (`spin.toml`) file.
+
+Now we have two separate Spin applications (`todo-app` and `todo-app-v2`) using their own labels to link to the same database. 
+
+Links and labels are application scoped. Whenever a specific Spin application (say `todo-app`) needs to access the database resource (`inspiring-otter`), it will do so via the label "data".
+
+The Spin application does not need to know the true name of the Cloud resource (`inspiring-otter` database) that it is interacting with. 
+
+A label is understood by all the components within a single application, but the labels applied do not hold any significance to other Spin applications. 
+
+> At this time, only NoOps SQL DB supports labels. However, in the future other Fermyon Cloud resources such as Key Value Store will support links and labels as well. 
+
+## Features of Links and Labels
+
+The benefits of using labels and Fermyon Cloud links include:
+
+* **Easy Sharing**: Share your Fermyon Cloud resource across applications effortlessly.
+* **Resource Creation Control**: You decide when to create and delete Fermyon Cloud resources.
+* **Decoupled Resource Lifecycle**: Allow Fermyon Cloud resources to have lifecycles independent of your Spin application's lifecycle.
+* **Seamless Cloud Integration**: Cloud experience smoothly integrates with local Spin development, no runtime configuration changes required. 
+* **Dynamic App Resource Configuration**: Change the data resource your application points to without rebuilding it.
+
+## Working With Links and Labels
+
+Whenever you add a string to your NoOps SQL database entry in the component manifest, you're creating a label. When you deploy your Spin application to Fermyon Cloud, you will be prompted to link the label to a specific NoOps SQL database instance. Later on, if you'd like to update the link you can do so via the `spin cloud link` command. 
+
```

Comment:
> Later on, if you'd like to update the link you can do so via the `spin cloud link` command.

We need to merge https://github.com/fermyon/cloud-plugin/pull/123 before the release to make this line true. cc @vdice

Reply:
Thanks @kate-goldenring 
#123 seems to be merged so we are good to go with this line.